src/CAN/ExpansionManager.cpp

```diff
@@ -24,6 +24,18 @@
 #define OBJECT_MODEL_FUNC(...) OBJECT_MODEL_FUNC_BODY(ExpansionManager, __VA_ARGS__)
 #define OBJECT_MODEL_FUNC_IF(...) OBJECT_MODEL_FUNC_IF_BODY(ExpansionManager, __VA_ARGS__)
 
+constexpr ObjectModelArrayTableEntry ExpansionManager::objectModelArrayTable[] =
+{
+	{
+		nullptr,					// no lock needed
+		[] (const ObjectModel *self, const ObjectExplorationContext& context) noexcept -> size_t { return 3; },
```

Comment:
How would you prefer array sizes are tracked?

Reply:
Move NumAccelerometerAxes (renamed from NumAxis) into an existing common header file so that you can use that value instead of 3.